NEWS.md

```diff
@@ -1,3 +1,47 @@
+# gsDesign2 1.1.3
+
+## Bug fixes
+
+- Fix the bug of `gs_design_rd()` when `info_scale = "h0_h1_info"` (#402, thanks to @LittleBeannie).
+- Fix the bug of `gs_spending_combo()` to enable HSD spending function (#444, thanks to @LittleBeannie).
+- Fix the bug of `fixed_design_maxcombo()` regarding the upper bounds (#445, thanks to @elong0527).
+- Fix the bug of `gs_design_wlr()` when the design is driven by information fraction only (#446, thanks to @LittleBeannie).
+- Fix the bug of `pw_info()` when there are many piecewise HRs (#460, thanks to @LittleBeannie).
+
+## Statistcial Improvements
+
+- The `gs_update_ahr` function is updated to 
+
+  + Enable unchanged futility bounds when it is fixed at the original design (#408, thanks to @LittleBeannie).
+  + Allow boundary updates when only certain interim analysis data is available (#436, thanks to @LittleBeannie).
+  + Allow boundary updates when alpha changes by keeping the same `info_scale` as the original design (#470, @LittleBeannie).
+  
+- Integer design (i.e., integer sample size and events) is updated to ensure exact integer sample size and events  (#452, #460, thanks to @LittleBeannie and @yihui).
+- Change the information fraction displayed at the summary-gt table from under H1 to H0 for logrank tests (#439, thanks to @LittleBeannie).
+- Add the sample size as the output of `ahr()` and `pw_info()` (#427, #433, thanks to @LittleBeannie).
+
+## User interface Improvements
+
+- Enable passing named vector of `col_decimals` and `analysis_decimals` to `summary.gs_design()` (#403, #431, @jdblischak).
+
+## Coding practive Improvements
+
+- Add robust check if `lower` is equivalent to `gs_b` (#413, thanks to @jdblischak )
+- The `summary()`, `as_gt()` `as_rtf()`, and `to_integer()` functions are refactored (#448, #449, #450, #465, #461, thanks to @yihui).
+- Remove the `full_alpha` argument from `as_rtf.gs_design()` (#458, thanks to @yihui).
+- Other coding practice improvement wroth mentioning (#409, #412, #429, #437, #440, #447, #453, #464, #467, $475, thanks to @yihui, @nanxstats, and @jdblischak).
+
+## Documentation
+
+- Update the documentation of `gs_b()` (#415, @jdblischak)
+
+## Testing
+
+- More Developer tests of `gs_power_ahr()` are added (#420, @LittleBeannie).
+- More Developer tests of `summery()` is added (#422, #426, thanks to @yuliasidi, @jdblischak and @LittleBeannie).
```

Comment:
```suggestion
- More Developer tests of `summary()` is added (#422, #426, thanks to @yuliasidi, @jdblischak and @LittleBeannie).
```

Reply:
Thank you so much for pointing out the 2 typos. They are corrected in the latest commit.